allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -195,7 +194,9 @@ def from_nwb(cls, nwbfile: NWBFile,
     @classmethod
     def from_data_file(cls, data_file: EyeTrackingFile,
                        sync_file: SyncFile,
-                       z_threshold: float = 3.0, dilation_frames: int = 2
+                       drop_frames: Optional[List[int]] = None,
+                       z_threshold: float = 3.0,
+                       dilation_frames: int = 2,
                        ) -> "EyeTrackingTable":
         """
```

Comment:
Please describe what `drop_frames` is for

Reply:
done